There's a bug in the configure script that thinks basename is not declared.
Normally this isn't a problem, but on Tiger the argument is "const char *"
which conflicts with the manually provided definition below. So this patch is
needed on Tiger only.

--- include/libiberty.h.orig
+++ include/libiberty.h
@@ -96,7 +96,7 @@
 #if !HAVE_DECL_BASENAME
   /* APPLE LOCAL basename */
 #if defined (__GNU_LIBRARY__ ) || defined (__linux__) || defined (__FreeBSD__) || defined (__OpenBSD__) || defined(__NetBSD__) || defined (__APPLE__) || defined (__CYGWIN__) || defined (__CYGWIN32__) || defined (__MINGW32__) || defined (HAVE_DECL_BASENAME)
-extern char *basename (char *);
+extern char *basename (const char *);
 #else
 /* Do not allow basename to be used if there is no prototype seen.  We
    either need to use the above prototype or have one from
